core/src/main/java/com/adobe/aem/commons/assetshare/components/details/Video.java

```diff
@@ -0,0 +1,19 @@
+package com.adobe.aem.commons.assetshare.components.details;
+
+/**
+ *
+ * Interface for Video Component
+ *
+ */
+public interface Video extends EmptyTextComponent {
```

Comment:
`@ConsumerType`  ... looks like we missed Consumer/Provider Type on most of these Details interfaces :T